system/lib/libc/musl/arch/emscripten/bits/fcntl.h

```diff
@@ -20,10 +20,11 @@
 #define O_NDELAY O_NONBLOCK
 
 #define F_DUPFD  0
-#define F_GETFD  1
-#define F_SETFD  2
-#define F_GETFL  3
-#define F_SETFL  4
+#define F_DUPFD_CLOEXEC 1
+#define F_GETFD  2
+#define F_SETFD  3
+#define F_GETFL  4
+#define F_SETFL  5
```

Comment:
Is this change needed?  I'd rather not change upstream musl sources unless we need to.   At least add a new flag seems orthogonal to fixing the bug, unless I'm missing something?

Reply:
Thanks for working on this BTW..  otherwise looks good!

Reply:
Nice catch. I tried it without this, and it turns out to be unnecessary. I believe I added that when I was trying to figure out a problem with missing C defines referenced in `src/library_syscall.js`. In the end, I just needed to add them to `struct_info.json`.

I've pushed a revision to remove this.